src/components/forms/RangeInput/RangeInput.tsx

```diff
@@ -17,47 +19,38 @@ type RangeInputProps = {
 export const RangeInput = ({
   className,
   inputRef,
+  textPreposition,
+  textUnit,
   ...inputProps
 }: RangeInputProps & JSX.IntrinsicElements['input']): React.ReactElement => {
   const classes = classnames('usa-range', className)
   // input range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
   const defaultMin = 0
   const defaultMax = 100
-  const { min, max, defaultValue } = inputProps
+  const defaultVal = Math.round(defaultMax / 2)
+  const defaultUnit = ''
+  const defaultPreposition = 'of'
+  const { min, max } = inputProps
   const rangeMin = min || defaultMin
   const rangeMax = max || defaultMax
-  const ariaMin = inputProps['aria-valuemin'] || rangeMin
-  const ariaMax = inputProps['aria-valuemax'] || rangeMax
-  const calculatedAriaValueNow =
-    inputProps['aria-valuenow'] ||
-    defaultValue ||
-    (rangeMax < rangeMin ? rangeMin : rangeMin + (rangeMax - rangeMin) / 2)
-  const convertValueType = (
-    value: string | number | readonly string[]
-  ): number | undefined => {
-    if (typeof value === 'number' || typeof value === 'string') {
-      return Number(value)
-    }
-    return undefined
-  }
-  const [ariaValue, setAriaValue] = React.useState<number | undefined>(
-    convertValueType(calculatedAriaValueNow)
-  )
-  const onValueChange: React.ChangeEventHandler<HTMLInputElement> = (e) => {
-    if (!inputProps['aria-valuenow']) setAriaValue(e.target.valueAsNumber)
-  }
+  const rangeUnit = textUnit || defaultUnit
+  const rangePreposition = textPreposition || defaultPreposition
+  const rangeValue = inputProps.defaultValue || defaultVal
```

Comment:
I'll make some renaming suggestions that I think will help with the value/defaultValue controlled/uncontrolled thing for future maintenance of the component.

We also probably want to check for undefined specifically here, since passing in `0` as the default value is `falsey`, but I think would be a legitimate thing to want to do

```suggestion
  const startingValue = defaultValue !== undefined ? defaultValue : defaultVal
```

Reply:
If we wanted to also accept `value` from input props as `valueProp`, we'd want that value used here as well:

```suggestion
  const startingValue =
    valueProp !== undefined
      ? valueProp
      : defaultValue !== undefined
      ? defaultValue
      : defaultVal
```